### ISSUE TITLE
How to set Hardware Concurrency from the Kameleo API?

### ISSUE THREAD

**Issue by JonathanR25** [kameleo-io/local-api-client-typescript#19]
What would be the proper code syntax form to set the Hardware Concurrency for a profile from the Kameleo API?

Also, could your provide some example code?

**Comment by MBailey01** [comment]
Here is an example that you can use to set the Hardware Concurrency for a profile through the API:

```
const createProfileRequest = BuilderForCreateProfile
  .forBaseProfile(chromeBaseProfileList[0].id)
  .setRecommendedDefaults()
  .setHardwareConcurrency('manual', 8) # Valid values: 1, 2, 4, 8, 12, 16.
  .build();
const profile = await client.createProfile({ body: createProfileRequest });
```

These answers are provided from the Kameleo [Set Hardware Concurrency from API](https://help.kameleo.io/hc/en-us/articles/15083372472092-Set-Hardware-Concurrency-from-API) article.

**Comment by KameleoStaff** [comment]
We recently released Kameleo v4.0.0, which introduced several changes and improvements to the platform. As a result, this code has been updated to be fully compatible with the latest version.

```
@type {import('@kameleo/local-api-client').CreateProfileRequest} */
const createProfileRequest = {
  fingerprintId: chromeBaseProfileList[0].id,
  hardwareConcurrency: {
    value: 'manual',  // Options: 'automatic', 'manual', 'off'
    extra: 8          // Valid values: 1, 2, 4, 8, 12, 16 (when 'manual' is selected)
  }
};

const profile = await client.profile.createProfile(createProfileRequest);
```

These answers are provided from the Kameleo [Set Hardware Concurrency from API](https://help.kameleo.io/hc/en-us/articles/15083372472092-Set-Hardware-Concurrency-from-API) article.
